src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -635,7 +635,7 @@ protected function parse(): void
         $this->template->assign('templates', $this->templates);
         $this->template->assign('positions', $this->positions);
         $this->template->assign('extrasData', json_encode(BackendExtensionsModel::getExtrasData()));
-        $this->template->assign('extrasById', json_encode(BackendExtensionsModel::getExtras()));
+        $this->template->assign('extrasById', json_encode($this->extras));
```

Comment:
The question is, can these extras be changed in between fetching them and assigning them here? 🤔

Reply:
well.. they can but they aren't. They're used to check if a page contains more than 1 block or if a block is placed on the homepage. Then they get assigned.

Reply:
Let's 🚢 then!